code/game/objects/structures/dresser.dm

```diff
@@ -18,65 +31,87 @@
 /obj/structure/dresser/deconstruct(disassembled = TRUE)
 	new /obj/item/stack/sheet/mineral/wood(drop_location(), 10)
 	qdel(src)
-
+/*
+//skyrat edit whole proc got modified pretty much
```

Comment:
Reference where the proc was moved to.

Reply:
This was all commented out. I'm gonna remove the confusing comment and changes.